app/project_tests.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
this should be removed in favor of the top-level one

Reply:
ah ok 👍